packages/wrangler/src/api/startDevWorker/DevEnv.ts

```diff
@@ -93,6 +95,34 @@ export class DevEnv extends EventEmitter {
 		});
 	}
 
+	/**
+	 * Awaits for all the local runtime controllers to be ready, and when they are
+	 * it resolves a promise on the proxy controller to let it know that the
+	 * local server is now ready to handle requests
+	 */
+	private async resolveAllRuntimeControllersReady(): Promise<void> {
+		await Promise.all(
+			this.runtimes
+				.filter(
+					(runtime) =>
+						runtime instanceof RemoteRuntimeController ||
+						runtime instanceof LocalRuntimeController
+				)
```

Comment:
Why are we filtering on runtime type here?

Reply:
good callout, this is a leftover from my previous iteration, I'll remove it 👍